src/content/roadmap/account-abstraction/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Account abstraction
+description: An overview of Ethereum's plans to make user accounts simpler and safer
+lang: en
+template: staking
```

Comment:
Will all pages use the staking template? 

I'm not 100% sure (build isn't working properly), but doesn't the template itself dictate what is contained in the navigation? If so, we'll need to account for this.

![Screenshot 2023-02-24 at 09 56 33](https://user-images.githubusercontent.com/62268199/221149140-302d35f8-89d0-45f3-938d-1d411ed5b37a.png)